src/backend/utils/mmgr/mcxt.c

```diff
@@ -756,6 +833,10 @@ MemoryContextStats_recur(MemoryContext topContext, MemoryContext rootContext,
 	 * collapse-able with other siblings. So, the siblingCount is set to 1.
 	 */
 	MemoryContext_LogContextStats(1 /* siblingCount */, allAllocatedTop, allFreedTop, currentAvailableTop, topContextName);
+#ifdef EXTRA_DYNAMIC_MEMORY_DEBUG
+	MemoryContext_printTopListOfChunks(chunks_htable);
+	chunks_htable = NULL;
```

Comment:
That and same lines confused me before I found hash_destroy call inside MemoryContext_printTopListOfChunks.
It possibly better to destroy and NULL in the same place.